docs/tutorials/setup.md

```diff
@@ -39,11 +42,11 @@ Here, we are running Ganache:
 
 This will make sure we have enough ETH and a high enough gas limit to deploy our larger contracts.
 
-If everything was setup correctly, we should be able to run automated tests from `protocol/core`. In a separate terminal, run the following commands from the `protocol/core` directory:
+If everything was setup correctly, we should be able to run automated tests from `protocol/packages/core`. In a separate terminal, run the following commands from the `protocol/packages/core` directory:
```

Comment:
nit: I know this was already phrased this way, but it's kinda weird that we're telling them to run `cd packages/core` _from_ the `packages/core` directory. I feel like we should either remove the `cd` or tell them to run the commands from root.

Reply:
yup. I agree. Seeing the code steps includes a `cd packages/core` I removed this from the main sentence. Should flow a bit more easily now.